packages/extension-chakra-storefront/src/components/register/index.jsx

```diff
@@ -41,7 +41,9 @@ const RegisterForm = ({submitForm, clickSignIn = noop, form}) => {
                 <Stack paddingTop={8} gap={8} paddingLeft={4} paddingRight={4}>
                     {form.formState.errors?.global && (
                         <Alert.Root status="error">
-                            <Alert.Indicator />
+                            <Alert.Indicator>
+                                <AlertIcon color="red.500" boxSize={4} />
+                            </Alert.Indicator>
                             <Alert.Description>
                                 {form.formState.errors.global.message}
                             </Alert.Description>
```

Comment:
We are missing migrating the Button`loading` prop to the equivalent `isLoading` for Chakra v3:

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/c39706f43d3b9540fd5ded303ae0719d1d73e212/packages/extension-chakra-storefront/src/components/register/index.jsx#L57

Reply:
🤔  `loading` prop is already correct, based on what I saw here https://www.chakra-ui.com/docs/components/button#loading

Reply:
You're right I was confused seeing other Button components using `isLoading`. It seems like `isLoading` still works in v3 but their docs ask to use `loading` instead 👍

Reply:
`isLoading` may still be accepted, but I'm not sure whether it still works.. or perhaps it does nothing.